### PR TITLE
Fixing #4

### LINKED ISSUE #4: Warning in Brent utility
There is a not initialized variable warning

https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/utilities/brent_iteration.h#L58

### PR DESCRIPTION
**Pull request by jcotela**
This PR fixes #4

### CODE REVIEW

**Comment by roigcarlo** [comment]
Looks good! :+1:

File: kratos/utilities/brent_iteration.h [hunk 1]
```diff
@@ -55,9 +55,9 @@ class BrentIteration
 
 		double c = b;
 		double fc = fb;
-		double d,p,q,r,s,tol1,xm;
+		double p,q,r,s,tol1,xm;
 		// Explicit initialization to silence a gcc compilation warning
-		double e = 0.0;
+		double d = 0.0, e = 0.0;
```
Comment (roigcarlo) [inline kratos/utilities/brent_iteration.h]: I would prefer to separate this in two lines

Comment (roigcarlo) [inline kratos/utilities/brent_iteration.h]: I would prefer to separate this

**Comment by pooyan-dadvand** [comment]
Ok for me!

**Comment by RiccardoRossi** [comment]
What happens if It is in am application?

Who can authorize the commit?

Riccardo

**Comment by pooyan-dadvand** [comment]
The admins of the repository. They can approve any part of the Kratos repo including core and apps.

**Pull request merged**